includes/definitions/extremeware.yaml

```diff
@@ -8,5 +8,6 @@ over:
     - { graph: device_processor, text: 'CPU Usage' }
     - { graph: device_mempool, text: 'Memory Usage' }
 discovery:
-    - sysObjectId:
-        - .1.3.6.1.4.1.1916.2
+    -
+      sysObjectId: .1.3.6.1.4.1.1916.2
+      sysDescr_regex: '/^XOS/'
```

Comment:
This should match no device right?  As you wrote it is looks for sysDescr that start with XOS.

Perhaps just
```
sysObjectId_regex: '/^.1.3.6.1.4.1.1916.2.1.([1-5]?[0-9]|6[0-5])$/'

```